oeps/processes/oep-0055-proc-project-maintainers.rst

```diff
@@ -0,0 +1,399 @@
+maintainer.  This might happen because:
+
+* A maintainer leaves the project or changes employment
+* A maintainer does not have the time or interest to continue in the
+  role
+* A maintainer is unreachable
+* A maintainer is not fulfilling the responsibilities of the role
+
+Trigger events for removing a maintainer would be:
+
+* A maintainer informs the project that they can no longer continue in
+  the role
+* A maintainer has been unreachable for more than 30 days
+* A maintainer has violated the project `Code of Conduct`_
+* A maintainer hasn't met project quality standards consistently
+  despite feedback
+
+In the case that a maintainer has violated the `Code of Conduct`_, the
+Goverance Team, specified in the Code of Conduct, will decide what the
+repercussions of the violation will be, including whether the violator
+can continue in the role of maintainer.  It should be noted that
+maintainers are expected to be exemplars and, as such, will be held to
+a high standard.
+
+In the case that a maintainer is not meeting project quality standards
+consistently, the scrum-of-scrums will inform the maintainer with the
+expectation that their components meet standards within a calendar
+month.  If, after a month, the quality issues are not resolved, the
+maintainer will be removed by the scrum-of-scrums.  Additionaly, a
+maintainer who is warned three (3) times in a single calendar year
+will also be removed.
+
+Documenting Maintainers
+-----------------------
+As it is essential that maintainers are discoverable and reachable,
+the person or persons responsible for maintaining a component should
+be listed in the repository README file along with the mechanism for
+contacting a maintainer.
+
+Jobs of the Maintainer
+----------------------
+Many of the day-to-day specifics of the maintainer role are outside of
+the scope of this document.  For example, here we will note that
+maintainers are responsible for ensuring that repositories meet
+project standards, but not the details of those standards.  Those
+standards will be codified in an OEP over time, but will initially be
+defined in the wiki page `requirements for public repositories`_ to
+allow rapid iteration.
+
+The key aspects of the maintainer role fall into the following
+categories
+
+* **Community Stewardship**: Maintainers are representatives of the
+  project in the community and are responsible for contributing to an
+  environment that is welcoming, vibrant, and growing.
+* **Project Management**: Maintainers are responsible for ensuring
+  that critical maintenance is done, but this doesn't mean doing it all
+  themselves. Effective delegation is an important maintainer skill.
+* **Quality Assurance**: Maintainers are responsible for the quality
+  of their components.  Quality will mean meeting defined project
+  standards.  It can also mean improving our standards or improving
+  our processes.
+* **Technical Vision**: Maintainers should have a documented technical
```

Comment:
Is this the only documentation we need?

Reply:
We'll certainly update our definitions over time, so I don't think this is carved in stone.

Do you have particular things in mind that you believe should be captured here?

Reply:
Will there be a standard place and/or format for the "technical vision" document(s)?

Reply:
I think so, but probably outside the scope of this OEP.  I think it would either need to be part of a standardized README or linked to from and standardized README.

Reply:
@schenedx I think this is tied with my above comment (L50 about appropriate documentation). What would you like to do with this and the above comment?

Reply:
@sarina I think you should list what you commented from above in this document here as an example. I really like how the documents at minimum should be the readme doc, but encourage other docs to exist as well on the repo. That example should be spelled out here so others can follow, even if they won't be matching the example 100%

Reply:
Thanks! Done!